### PR TITLE
Rebase onto Element iOS 1.8.3 tag

### PR DESCRIPTION
**Pull request by phlniji**
#461

### CODE REVIEW

File: Riot/Model/HomeserverConfiguration/HomeserverConfigurationBuilder.swift [hunk 1]
```diff
@@ -61,24 +61,24 @@ final class HomeserverConfigurationBuilder: NSObject {
...
-        let tileServerConfiguration = HomeserverTileServerConfiguration(mapStyleURL: tileServerMapStyleURL)
+//        let tileServerMapStyleURL: URL
+//        if let mapStyleURLString = wellKnown?.tileServer?.mapStyleURLString,
+//           let mapStyleURL = URL(string: mapStyleURLString) {
+//            tileServerMapStyleURL = mapStyleURL
+//        } else {
+//            tileServerMapStyleURL = BuildSettings.tileServerMapStyleURL
+//        }
+//        
+//        let tileServerConfiguration = HomeserverTileServerConfiguration(mapStyleURL: tileServerMapStyleURL)
```
Comment (yostyle) [inline Riot/Model/HomeserverConfiguration/HomeserverConfigurationBuilder.swift:72]: You should add a comment prefixed by "// Tchap: " to explain this update

**Comment by yostyle** [comment]
LGTM

**Comment by giomfo** [comment]
@phlniji during my tests:
- [ ]  the favourites screen doesn't load anymore (I observed an infinite loading wheel)
- [ ] invite by email a new user failed (I used the button 
image

**Review by giomfo — APPROVED** [review]
@phlniji I approve your rebase, I let you just reply to my 2 remarks [here](https://github.com/tchapgouv/tchap-ios/pull/474#issuecomment-1064080057) before merging. Create a dedicated issue is possible to track them if they cannot be fixed easily

**Pull request merged**